medusa/providers/torrent/json/t411.py

```diff
@@ -205,6 +216,8 @@ def search(self, search_strings, age=0, ep_obj=None):
         if not self.login():
             return results
 
+        self._get_terms()
```

Comment:
This may be a test but you were talking about getting it released for 0.1.12.  To do that it needs a proper review now or pushed to 0.1.13.